llm/src/speed_monitor_w_mfu.py

```diff
@@ -61,11 +61,7 @@ def get_gpu_flops_available(state):
         except:
             gpu_flops_available = None
 
-    if gpu_flops_available is not None:
-        print(
-            f'Using {gpu_flops_available=} when calculate MFU (assumes {state.precision.value} precision using {torch.cuda.get_device_name()}).'
-        )
-    else:
```

Comment:
@vchiley the reason I'm removing this for now is b/c it was printing out after the first batch of each run and it looks kinda messy for demos